actions/src/create-cut-pr/main.test.ts

```diff
@@ -0,0 +1,397 @@
+import type { OctokitResponse } from "@octokit/types";
+import { describe, afterEach, beforeEach, it, vi } from "vitest";
+
+import { getNextCutVersion, run } from "./main";
+
+import { changelog, type Ctx } from "@/lib";
+import {
+  Repository,
+  type GithubRepository,
+  type Octokit,
+  type PullRequest,
+} from "@/lib/github";
+import {
+  CHANGELOG_LABEL,
+  MAJOR_SEVERITY_LABEL,
+  MINOR_SEVERITY_LABEL,
+} from "@/lib/labels";
+import { asyncIterable, mockPr } from "@/lib/test-utils";
+
+describe("create-cut-pr", () => {
+  let ctx: Ctx;
+
+  beforeEach(() => {
+    ctx = {
+      context: {
+        refName: "main",
+      },
+      git: {
+        mainBranch: "main",
+        configUser: vi.fn(),
+        checkout: vi.fn(),
+        createBranch: vi.fn(),
+        moveBranch: vi.fn(),
+        commit: vi.fn(),
+        push: vi.fn(),
+        fetch: vi.fn(),
+      },
+      repo: {
+        defaultBranch: "main",
+        pullRequests: vi.fn().mockReturnValue(asyncIterable([])),
+        branches: vi.fn().mockReturnValue(asyncIterable([])),
+        createPullRequest: vi.fn(),
+      },
+      exec: vi.fn(),
+      execOutput: vi.fn(),
+    } as unknown as Ctx;
+  });
+
+  describe.for([
+    ["with no other release branches", [], "0.0", "0.0"],
+    ["with one other release branch", ["release/1.0"], "1.1", "2.0"],
+    [
+      "with multiple other release branch",
```

Comment:
english police is here
```suggestion
      "with multiple other release branches",
```